docs/sources/mimir/configure/configuration-parameters/index.md

```diff
@@ -4825,6 +4825,10 @@ The `memcached` block configures the Memcached-based caching backend. The suppor
 ```
 
 ### redis
+{{< admonition type="caution" >}}
+Starting with Mimir 2.14, Redis-based caching is deprecated.
+It will be removed in a future release.
+{{< /admonition >}}
```

Comment:
2.14 is planned to be release in 1+ week, with the rc.0 today. I'm not sure, I would rush in marking everything about redis in the code as deprecated to support this. Can we deprecate it in 2.15?

Reply:
We want to deprecated it in 2.14 to give people as much time to migrate as possible.